utils/fwutils.py

```diff
@@ -127,6 +139,19 @@ def reflash_ipu(self) -> None:
         else:
             self.logger.info("Skipping flash_spi_image")
 
+        self.logger.info("Step 5: apply_fixboard")
+        if self.should_run("apply_fixboard"):
+            should_fixboard = self.fixboard_is_needed()
+            if should_fixboard:
+                self.logger.info("Applying fixboard!")
+                self.apply_fixboard()
+            elif should_fixboard is None:
```

Comment:
Let's get rid of this (see other comment)

should_fixboard should just return true, false, or throw an exception if something unexpected happens.